docs/source/mayavi/sphinxext/sphinxext/comment_eater.py

```diff
@@ -1,10 +1,7 @@
 from __future__ import division, absolute_import, print_function
 
 import sys
-if sys.version_info[0] >= 3:
-    from io import StringIO
-else:
-    from io import StringIO
+from io import StringIO
```

Comment:
These files are copied over from http://github.com/numpy/numpydoc
not sure what is wrong.